nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -173,13 +107,24 @@ let
     # Menu configuration
     #
 
-    insmod gfxterm
-    insmod png
+    # insmod gfxterm
+    # insmod png
+
+    # TODO: fix this (re-load location where modules get pulled from)
+    insmod /boot/grub/i386-pc/font.mod
```

Comment:
when doing for ex insmod font, it complains about /boot/grub/i386-pc/font.mod not being found
If I give it the full path, it works

This is because the modules are on the .iso not the geenrated standalone .img, though really they SHOULD be pulled from the current root, which is changed in grub.cfg with search --set=root...

Reply:
set prefix=($root)... fixed it